cmd/apps/registry_app.go

```diff
@@ -24,8 +24,8 @@ import (
 func MakeInstallRegistry() *cobra.Command {
 	var registry = &cobra.Command{
 		Use:          "docker-registry",
-		Short:        "Install a Docker registry",
-		Long:         `Install a Docker registry`,
+		Short:        "Install a community maintained Docker registry chart",
+		Long:         `Install a community maintained Docker registry chart`,
```

Comment:
I'd like to see the URL in the long description - wrapped as best as possible to 80 chars

Reply:
Link also to the Open Source Docker registry - https://github.com/distribution/distribution

Reply:
@Shikachuu PTAL when you can

Reply:
Done :heavy_check_mark: